ui/desktop/electron-app/src/helpers/app-updater.js

```diff
@@ -33,12 +33,12 @@ const findLatestVersion = (url) => {
 
 // Find zip archive for update
 const findUpdateArchive = (version) => {
-  const url = `${releasesUrl}boundary-desktop_${version}/boundary-desktop_${version}_darwin_amd64.zip`;
+  const url = `${releasesUrl}${version}/boundary-desktop_${version}_darwin_amd64.zip`;
```

Comment:
This comment is out of topic, but at some point we might need to make the architecture `_darwin_amd64` dynamic. I am thinking on windows client o M1 apple chips (arm64)

Reply:
Agree. Incoming windows support PR configures release url based on os platform and architecture.

Reply:
Awesome!